crates/geyser-rabbitmq/src/plugin.rs

```diff
@@ -78,8 +78,42 @@ impl GeyserPlugin for GeyserPluginRabbitMq {
 
     fn on_load(&mut self, cfg: &str) -> Result<()> {
         solana_logger::setup_with_default("info");
+
         let metrics = Metrics::new_rc();
 
+        let version;
+        let host;
+
+        {
+            let ver = env!("CARGO_PKG_VERSION");
+            let git = option_env!("META_GIT_HEAD");
+            // let rem = option_env!("META_GIT_REMOTE");
+
+            {
+                use std::fmt::Write;
+
+                let mut s = format!("v{}", ver);
+
+                if let Some(git) = git {
+                    write!(s, "+git.{}", git).unwrap();
+                }
+
+                version = s;
+            }
+
+            // let rustc_ver = env!("META_RUSTC_VERSION");
+            // let build_host = env!("META_BUILD_HOST");
+            // let target = env!("META_BUILD_TARGET");
+            // let profile = env!("META_BUILD_PROFILE");
+            // let features = env!("META_BUILD_FEATURES");
```

Comment:
You got these. Want to delete the comments?

Reply:
They're produced in the build.rs and passed at build-time to these variables, but I don't use 'em at the moment.  Plan is to send more detailed telemetry in the future.